### DISCUSSION TITLE
YouTube Rate Limits (Too Many Requests)

### DISCUSSION THREAD

**Discussion by AaronPhilipp** [jdepoix/youtube-transcript-api#176] [Q&A]
Hi,

When I want to get the transcript for multiple Videos I get the following Error:

```
youtube_transcript_api._errors.YouTubeRequestFailed: 
Could not retrieve a transcript for the video https://www.youtube.com/watch?v=429 Client Error: Too Many Requests for url: XXX This is most likely caused by: XXX
```

Do you have an idea how to solve the problem or when Google allows me again to retrieve transcripts?

Thank you in advance!

**Comment by jdepoix** [comment]
Hi @AaronPhilipp,
you could have a look at #60 and #74 to follow some of the discussion people had around this topic. But the TLDR is: there unfortunately is not much you can do, except for using a different IP. There also is no definite answer to the amount of requests you can do before being rate limited, nor the time you are banned for. Users have reported very different numbers and it seems to me that YouTube limits usage dynamically depending on load.

**Reply by jdepoix** [reply]
There now is an [entire section in the README](https://github.com/jdepoix/youtube-transcript-api#working-around-ip-bans-requestblocked-or-ipblocked-exception) that tells you how you can work around IP bans! 🙂

**Reply by lovehumans** [reply]
Thanks but this project is dead to me, now that I can't even use yt-dlp w/o providing my cookies anymore.
